cypress/integration/26-register_registration/creator_begins_registering_a_registration/begin_registering_registration.js

```diff
@@ -0,0 +1,148 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';

```

Comment:
er det et nivå for mye her? skulle ikke filnavnet vært `creator_begins_registering_a_registration.js`?

Reply:
Nei, det stemmer det. Forkorter navnene på javascriptfilene så de ikke blir så lange. De må ligge i en katalog som heter det samme som featuren.